wolfcrypt/src/rsa.c

```diff
@@ -267,12 +267,19 @@ static void wc_RsaCleanup(RsaKey* key)
 
 int wc_InitRsaKey_ex(RsaKey* key, void* heap, int devId)
 {
-    int ret = 0;
+    int ret      = 0;
+    int isPkcs11 = 0;
```

Comment:
Please wrap all `isPkcs11` in `HAVE_PKCS11`. I don't want to grow code size or have to have an "unused"